src/main/java/core/basesyntax/file/FileReaderImpl.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax.file;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FileReaderImpl implements FileReader {
+
+    @Override
+    public List<String> read(String filePath) {
+        List<String> lines = new ArrayList<>();
+        try (BufferedReader reader = Files.newBufferedReader(Paths.get(filePath))) {
+            String line;
+            boolean isFirstLine = true;
+            while ((line = reader.readLine()) != null) {
+                if (isFirstLine) {
+                    isFirstLine = false;
+                    continue;
+                }
```

Comment:
This service should only read the file, without any additional logic. Move it to the DataConverter